lib/datastore/entity.js

```diff
@@ -314,53 +332,53 @@ module.exports.isKeyComplete = function(key) {
  *
  * @example
  * propertyToValue({
- *   boolean_value: false
+ *   booleanValue: false
  * });
  * // false
  *
  * propertyToValue({
- *   string_value: 'Hi'
+ *   stringValue: 'Hi'
  * });
  * // 'Hi'
  *
  * propertyToValue({
- *   blob_value: new Buffer('68656c6c6f')
+ *   blobValue: new Buffer('68656c6c6f')
  * });
  * // <Buffer 68 65 6c 6c 6f>
  */
 function propertyToValue(property) {
```

Comment:
Renamed to `decodeValueProto`: https://github.com/GoogleCloudPlatform/gcloud-node/pull/897#discussion_r41195136